lib/stripe.ts

```diff
@@ -0,0 +1,48 @@
+import { createStripeTeam, getByTeamId } from 'models/stripeTeam';
+import Stripe from 'stripe';
+import env from '@/lib/env';
+
+export const stripe = new Stripe(env.stripe.stripeSecretKey ?? '', {
+  // https://github.com/stripe/stripe-node#configuration
+  apiVersion: '2022-11-15',
+  // Register this as an official Stripe plugin.
+  // https://stripe.com/docs/building-plugins#setappinfo
+  appInfo: {
+    name: 'saas-starter-kit',
+    version: '0.1.0',
+  },
+});
+
+export const getURL = () => {
```

Comment:
Can we directly use the existing `appUrl`, the value would be `https` on production.